### PR TITLE
Update avalanche-bridge-faq.md

### PR DESCRIPTION
**Pull request by salwao2**
Updated with two new q&a. Responses posted by validator on Discord.

### CODE REVIEW

File: learn/avalanche-bridge-faq.md [hunk 1]
```diff
@@ -120,6 +120,11 @@ You can use Metamask’s SWAP function to swap from ETH to WETH. Alternatively,
...
 
+#### Why doesn’t the newly bridged token appear on the portfolio automatically?
+
+It can't appear automatically because there are too many contracts running on the EVM. Metamask can't scan all of them to check if your address has a balance. Tokens are not physically in your C-chain address. They are "inside" the smart contract related to the token, and it's written in the chain that your address has xxx token. Your wallet (AVAX or Metamask) has to check all those contracts in order to see if there are some token belonging to you. But there are so many contracts it will take a lot of time, thus you have to manually tell to your wallet which contract it has to check.
+
```
Comment (yulin-dong) [inline learn/avalanche-bridge-faq.md:126]: @jpop32 can you review this?

**Comment by danlaine** [comment]
In one answer, we say that token balances are held in smart contracts, but in the other, we say that balances aren't held in a smart contract. Which is it?

File: learn/avalanche-bridge-faq.md (hunk 1 above)
Comment (jpop32) [reply learn/avalanche-bridge-faq.md:126]: Technically correct, but the wallet can check the common tokens, like the ones used by the bridge. It currently does so for the AEB tokens, I expect it makes sense to switch it over to AB tokens then. Emre can probably provide authoritative info.

File: learn/avalanche-bridge-faq.md [hunk 2]
```diff
@@ -46,6 +46,12 @@ Yes, you can click on the “Speed Up” button on Metamask. “Speeding up” a
 
 When transferring from Avalanche to Ethereum, Metamask shows that 0 tokens are to be transferred, not the actual number of tokens. This is a known problem with Metamask.
 
+#### What are the bridge contract adresses? 
+
+Bridge Addresses:
+    - ETH: `0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0` https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0
+    - AVAX: `0x50Ff3B278fCC70ec7A9465063d68029AB460eA04` https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04
+
```
Comment (yulin-dong) [inline learn/avalanche-bridge-faq.md:54]:
```suggestion
Bridge Addresses:
    - ETH: `0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0` [https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0](https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0)
    - AVAX: `0x50Ff3B278fCC70ec7A9465063d68029AB460eA04` [https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04](https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04)
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: learn/avalanche-bridge-faq.md (hunk 2 above)
Comment (yulin-dong) [reply learn/avalanche-bridge-faq.md:54]: This will make the link clickable.

File: learn/avalanche-bridge-faq.md (hunk 1 above)
Comment (yulin-dong) [reply learn/avalanche-bridge-faq.md:132]: @kanatliemre can you help, please?

File: learn/avalanche-bridge-faq.md [hunk 3]
```diff
@@ -46,6 +46,14 @@ Yes, you can click on the “Speed Up” button on Metamask. “Speeding up” a
 
 When transferring from Avalanche to Ethereum, Metamask shows that 0 tokens are to be transferred, not the actual number of tokens. This is a known problem with Metamask.
 
+#### What are the bridge contract adresses? 
+
+Bridge Addresses:
+    - ETH: `0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0` https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0
+    - AVAX: `0x50Ff3B278fCC70ec7A9465063d68029AB460eA04` https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04
```
Comment (yulin-dong) [inline learn/avalanche-bridge-faq.md:53]: @salwao2 it is weird that github made this mess. Can you remove the lines 52-53?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: learn/avalanche-bridge-faq.md (hunk 1 above)
Comment (kanatliemre) [reply learn/avalanche-bridge-faq.md:134]: Yes the web wallet (not metamask), has a default list of tokens supported out of the box. We can add the new bridge tokens there. Is there a tokenlist url for these tokens?

Comment (yulin-dong) [reply learn/avalanche-bridge-faq.md:132]: Token list: https://github.com/ava-labs/avalanche-bridge-resources/blob/main/token_list.json

**Comment by salwao2** [comment]
> In one answer, we say that token balances are held in smart contracts, but in the other, we say that balances aren't held in a smart contract. Which is it?

**Comment by jpop32** [comment]
> In one answer, we say that token balances are held in smart contracts, but in the other, we say that balances aren't held in a smart contract. Which is it?

Both are true.

We're talking about two different things here: how the bridge operates, and how the tokens operate.

Bridge doesn't hold the tokens in a 'bridge contract' (like how AEB did it, for example). Address that holds the bridge tokens is not a contract but a simple address (wallet). That's what one answer refers to.

Other thing is, ERC-20 token balances are always (bridge or otherwise) written and maintained by the contract that created a particular token. That's why you need to query token contract to find out the balance, and why Metamask needs to do the work to find the token balances, which is what the other answer refers to.

Hope this clears things up a bit.

**Comment by danlaine** [comment]
Closing in favor of #240 because this PR is from Salwa's fork so we can't edit it directly

**Pull request closed without merging**